src/com/projectkorra/projectkorra/earthbending/combo/EarthPillars.java

```diff
@@ -100,8 +101,7 @@ public void progress() {
 					if (lent instanceof Player && !((Player) lent).isOnline()) {
 						continue;
 					}
-
-					lent.setVelocity(new Vector(0, this.knockup, 0));
+					GeneralMethods.setVelocity(this,lent, new Vector(0, this.knockup, 0));
```

Comment:
Please change to `GeneralMethods.setVelocity(this, lent, new Vector(0, this.knockup, 0));`